valohai/internals/utils.py

```diff
@@ -6,3 +6,15 @@ def string_to_bool(value: str) -> bool:
     if value.lower() == "false":
         return False
     return True
+
+
+def check_sha256_hashing(filepath: str) -> str:
```

Comment:
This function could maybe use a better name – it isn't checking anything, it's just computing the hash of the given file?